helm/flowforge/templates/emqx.yaml

```
@@ -0,0 +1,250 @@
+                        websocket {
+                            mqtt_path = "/"
+                            allow_origin_absence = true
+                            check_origin_enable = false
+                            fail_if_no_subprotocol = true
+                            supported_subprotocols = "mqtt, mqtt-v3, mqtt-v3.1.1 mqtt-v5"
+                            mqtt_piggyback = multiple
+                            compress = false
+                            idle_timeout = 7200s
+                            max_frame_size = infinity
+                            proxy_address_header = "x-forwarded-for"
+                            proxy_port_header = "x-forwarded-port"
+                        }
+                    }
+                }
+            }
+            api_key {
+              bootstrap_file = "/mounted/config/api-keys"
+            }
+    coreTemplate:
+        spec:
+            {{- if .Values.forge.registrySecrets }}
+            imagePullSecrets:
+            {{- range .Values.forge.registrySecrets }}
+              - name: {{ . }}
+            {{- end }}
+            {{- end }}
+            env:
+              - name: EMQX_DASHBOARD__DEFAULT_PASSWORD
+                valueFrom:
+                    secretKeyRef:
+                        name: emqx-config-secrets
+                        key: EMQX_DASHBOARD__DEFAULT_PASSWORD
+            volumeClaimTemplates:
+                {{- if .Values.broker.storageClassName }}
+                storageClassName: {{ .Values.broker.storageClassName }}
+                {{- end}}
+                resources:
+                    requests:
+                        storage: 5Gi
+                accessModes:
+                    - ReadWriteOnce
+            extraVolumes:
+              - name: config
+                secret:
+                    secretName: emqx-config-secrets
+            extraVolumeMounts:
+              - name: config
+                mountPath: /mounted/config/api-keys
+                subPath: api-keys
+            {{- if .Values.forge.broker.affinity }}
+            affinity: {{ toYaml .Values.forge.broker.affinity | indent 12 }}
+            {{- end }}
+            {{- if .Values.forge.broker.tolerations}}
+            tolerations:
+            {{ toYaml .Values.forge.broker.tolerations | nindent 12 }}
+            {{- end }}
+    listenersServiceTemplate:
+        spec:
+        {{- if .Values.broker.listenersServiceTemplate }}
+{{ toYaml .Values.broker.listenersServiceTemplate | indent 12 }}
+        {{ else }}
+            type: ClusterIP
+        {{- end }}
+    dashboardServiceTemplate:
+        spec:
+        {{- if .Values.broker.dashboardServiceTemplate }}
+{{ toYaml .Values.broker.dashboardServiceTemplate | indent 12 }}
+        {{ else }}
+            type: ClusterIP
+        {{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+    name: emqx-config-secrets
+    namespace: {{ .Release.Namespace }}
+type: Opaque
+stringData:
```

```suggestion
data:
    EMQX_DASHBOARD__DEFAULT_PASSWORD: {{ "topSecret" | b64enc | quote }}
     api-keys: |
         {{ "flowfuse:verySecret:administrator" | b64enc | quote }} 
```

How about this? I found `stringData` causing issues. In the background, kube "transforms" stringData to data anyway.